notebooks/236-stable-diffusion-v2/236-stable-diffusion-v2-text-to-image.ipynb

```diff
@@ -2,10 +2,8 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.        compressed_unet = nncf.compress_weights(unet)
</pre><p>For the latest NNCF version <code>ignored_scope=nncf.IgnoredScope(types=['Convolution'])</code> should be added</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Added</p>